components/topbar/user/src/index.js

```diff
@@ -52,17 +51,25 @@ class TopbarUser extends Component {
   /**
    * Lock body element scroll.
    */
-  _lockBodyScroll () {
-    window.document.documentElement.style = HTML_DISABLED_SCROLL_STYLE
-    window.document.body.style = BODY_DISABLED_SCROLL_STYLE
+  _lockBodyScroll = () => {
+    this._verticalScrollPosition = window.scrollY
+    window.document.documentElement.classList.add('html-has-scroll-disabled')
```

Comment:
Why is 'html-has-scroll-disabled' not a constant as HTML_DISABLED_SCROLL_STYLE was?
If a string is repeated, I usually use constant for that.

Reply:
Sure 👍